cuisino/evaluation/templates/evaluation/main.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
```

Comment:
these html helps me understand what you were trying to get at. i will use our template to make adjustments. can leave this be.